common/client-libs/validator-client/src/nymd/mod.rs

```diff
@@ -198,6 +198,26 @@ impl<C> NymdClient<C> {
             .await
     }
 
+    pub async fn get_total_mix_stake(&self) -> Result<Uint128, NymdError>
+    where
+        C: CosmWasmClient + Sync,
+    {
+        let request = QueryMsg::GetTotalMixStake {};
+        self.client
+            .query_contract_smart(self.contract_address()?, &request)
+            .await
+    }
+
+    pub async fn get_total_gt_stake(&self) -> Result<Uint128, NymdError>
+    where
+        C: CosmWasmClient + Sync,
+    {
+        let request = QueryMsg::GetTotalGtStake {};
```

Comment:
And it seems same thing for the `QueryMsg::GetTotalGtStake` if we're renaming things